nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/PutEmail.java

```diff
@@ -437,19 +431,18 @@ public void process(InputStream in) throws IOException {
      */
     private Session createMailSession(final Properties properties) {
         String authValue = properties.getProperty("mail.smtp.auth");
```

Comment:
I notice a few other local variables that are not final but could be.  What do you think about making them final in this PR?